distribution/docs/src/main/resources/_contents/_configuring/configuring-from-configuration-files-contents.adoc

```diff
@@ -344,6 +345,29 @@ application/x-msdownload,
 application/x-msmetafile
 |Yes
 
+6+^h|Solr Provider Properties
+
+|Solr Client
+|solr.client
+|String
+|Solr Catalog Provider selected. See <<_configuring_thread_pools,Configuring Thread Pools>>
+|HttpSolrClient, CloudSolrClient, EmbeddedSolrServer
+|Default
+
+|Solr Catalog Provider Url
+|solr.http.url
+|String
+|Solr Catalog Provider Url (can be either a Standalone Solr Server or a Cluster of Standalone Solr Servers)
```

Comment:
I think what @figliold meant was that the `solr.http.url` property is only used for a Standalone Solr Server. So what's in the parentheses here is a little off since this property is not needed for a cluster of Standalone Servers (a.k.a Solr Cloud).